src/__tests__/react-plotly.test.js

```diff
@@ -44,10 +44,10 @@ describe('<Plotly/>', () => {
     });
 
     describe('initialization', function() {
-      test('calls Plotly.newPlot on instantiation', done => {
+      test('calls Plotly.react on instantiation', done => {
```

Comment:
you know, I forgot we have minimal tests for this thing... Could you please add some tests for the logic in https://github.com/plotly/react-plotly.js/pull/126 please? :)

Reply:
(you can add the tests in a separate PR, but we should get them in)

Reply:
Done